themes/mainroad/assets/css/style.css

```diff
@@ -1375,6 +1363,10 @@ summary:hover {
 	--color-highlight: #d7742f;
 	--color-badges-beta: #d7742f;
 	--color-badges-new: #e57;
+	--color-badges-legacy: #13cace;
```

Comment:
The contrast between the background of the badge and the text color in dark mode (white) is bad.

![image](https://github.com/neoforged/websites/assets/21304337/e99ec184-5aa6-48de-89b3-e21a20ef8144)

Reply:
![image](https://github.com/neoforged/websites/assets/140878474/c193ea6c-84a5-4844-b315-32c2ca400edc)
I've changed the legacy badge's color in dark mode. _(btw i think these version numbers would look good)_

Reply:
![image](https://github.com/neoforged/websites/assets/140878474/edb6a517-f660-4bd1-8cb4-4c4ccbe3ac19)
this does not look good here...